### DISCUSSION TITLE
Add failure detection API to monaco

### DISCUSSION THREAD

**Discussion by chanusch** [Dynatrace/dynatrace-configuration-as-code#278] [Archive] [answered]
It would be great to support creating ParameterSets and Rules of the failure detection api. (Configuration API v1)

This would be 
`PUT api/config/v1/service/failureDetection/parameterSelection/parameterSets/{id}`
and
`PUT api/config/v1/service/failureDetection/parameterSelection/rules/{id}`

We have to make those steps by hand at the moment and it would be great to support this in monaco.

**Comment by patrickpichler** [comment, marked as answer]
Awesome idea :+1: 

The issue for this is #280
